great_expectations/rule_based_profiler/data_assistant/data_assistant_runner.py

```diff
@@ -91,6 +91,7 @@ def run_impl(self) -> Callable:
 
         def run(
             batch_request: Optional[Union[BatchRequestBase, dict]] = None,
+            exact_estimation: bool = True,
```

Comment:
I'm not ready to decide this yet :( It feels strange that in order to use an alternative method (like Bootstrap) that we have to set `exact_estimation = False`. Alternatives could be some kind of mode flag / enum (also problematic), or that we have a more semantic parameter, such as `is_data_valid = True` that means we use an exact estimator

Reply:
@donaldheppner I agree with you concern.  Specifically, I also feel it is odd that in order to get to default estimator, which is currently probabilistic, we have to set some flag to False.  I have another idea.  How about we simply make the default estimator in the `OnboardingDataAssistant` -- in all of its Rules equal to "exact".  We can then turn the present implementation of `exact_estimation` around into the global estimator value, which can be any estimator we support.  This is a string value, just like we use today in every Rule.  I feel good about this, because `exact` is good globally for starters, and `bootstrap` is good globally for continued expectation authoring and productionalizing the suite.  Please let me know what you think.  Thank you! /cc @talagluck @Shinnnyshinshin @NathanFarmer

Reply:
@donaldheppner All changes we discussed have been committed.  Please re-review.  Thank you.